packages/mdc-select/mdc-select.scss

```diff
@@ -95,6 +96,12 @@
     white-space: nowrap;
     overflow: hidden;
   }
+
+  option.mdc-list-item {
```

Comment:
Why have the `option` selector on this at all?

Reply:
Because it is not necessary to override the option background in a 'js select' like ```li.mdc-list-item```. His parent ```.mdc-theme--dark .mdc-simple-menu``` already has this background.

Reply:
Lets take a step back... Is this even doing anything?

Regardless we want to avoid tag selectors at all cost.